src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -1,10 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>System.Net.NetworkInformation</AssemblyName>
     <OutputType>Library</OutputType>
     <ProjectGuid>{3CA89D6C-F8D1-4813-9775-F8D249686E31}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+    <!-- System.Net.Sockets.SafeSocketHandle is defined both in System.Net.Sockets (public)
+         and System.Net.NetworkInformation (internal). -->
+    <NoWarn>$(NoWarn);CS0436</NoWarn>
```

Comment:
Would you mind opening an issue for this, and then include the issue number here in the comment?  We should be able to remove the private implementation in NetworkInformation.

Reply:
#32944